### DISCUSSION TITLE
Automa extension is will end in chrome?

### DISCUSSION THREAD

**Discussion by MarclHenrique** [AutomaApp/automa#1794] [Q&A]
The availability of this extension may soon end because it does not follow best practices for Chrome extensions.

![Captura de tela 2024-07-30 113904](https://github.com/user-attachments/assets/b9072bcf-10f4-4c70-b330-4265fc45aa2f)

**Comment by 3Deyedoc** [comment]
I hope not. I just started using this extension. I would hate for it to go away just as I get used to using it. Does anyone know if Chrome will remove it or will it just give us an error in perpetuity? Better yet, will it get updated so we can keep using it?

**Comment by jmdone** [comment]
With this in mind, it would be nice to know which Google Chrome Extension best practices are not being followed.   A warning like this could be for something minor like using the wrong manifest format, or the issue is of a major concern and could compromise end user data privacy.

Would love to hear what needs to be corrected.

**Reply by jmdone** [reply]
That said, I don't see the same warning on the US / English site.

**Reply by jmdone** [reply]
One reason is likely due to the fact this extension is still using Manifest V2 instead of V3 as required by [Google's Extensions Best Practices](https://developer.chrome.com/docs/webstore/best-practices#manifest_version_3)

![image](https://github.com/user-attachments/assets/94a8e7a5-cbb9-4d89-a930-69e97cb93b2c)

![image](https://github.com/user-attachments/assets/fc5cdabb-4dfb-4621-9085-d8180c2c00d9)

**Comment by 3Deyedoc** [comment]
I think manifest V2 is the issue. I'm not sure where the upgrade is at. I see a blog post from '22 talking about migration to V3 but nothing after that.

**Reply by nkrusch** [reply]
Here is the official migration guide: https://developer.chrome.com/docs/extensions/develop/migrate

The big change is using a service worker and how that affects the extension's background context.
